proto/transport.proto

```diff
@@ -0,0 +1,265 @@
+service TransportApi {
+  rpc GetTransport(GetTransportRequest) returns (Transport);
+  rpc PullTransport(PullTransportRequest) returns (stream PullTransportResponse);
+}
+
+service TransportInfo {
+  rpc DescribeTransport(DescribeTransportRequest) returns (TransportSupport);
+}
+
+// Transport describes the current state of the transport.
+message Transport {
+    enum Direction {
+      DIRECTION_UNSPECIFIED = 0;
+      // i.e. the device is stationary and not moving in any direction.
+      NO_DIRECTION = 1;
+      UP = 2;
+      DOWN = 3;
+      IN = 4;
+      OUT = 5;
+      CLOCKWISE = 6;
+      ANTICLOCKWISE = 7;
+      FORWARD = 8;
+      BACKWARD = 9;
+      EAST = 10;
+      WEST = 11;
+      NORTH = 12;
+      SOUTH = 13;
+      // Left and right are relative to the intended forwards direction of the transport.
+      // If this is not known use SIDEWAYS.
+      LEFT = 14;
+      RIGHT = 15;
+      SIDEWAYS = 16;
+    }
+
+    enum OperatingMode {
+      OPERATING_MODE_UNSPECIFIED = 0;
+      NORMAL = 1;
+      // The transport is in service or inspection control.
+      SERVICE_CONTROL = 2;
+      // The transport is under a firefighter transport or firefighting transport mode of control. This usually happens
+      // during a fire alarm when firemen are on the site.
+      FIREFIGHTER_CONTROL = 3;
+      OUT_OF_SERVICE = 4;
+      // The transport is being operated by emergency power and may be in limited operation.
+      EMERGENCY_POWER = 5;
+      // The transport will not respond to any instruction until the VIP has reached their destination.
+      VIP_CONTROL = 6;
+      EARTHQUAKE_OPERATION = 7;
+      FIRE_OPERATION = 8;
+      // The transport is being manually operated by an attendant.
+      ATTENDANT_CONTROL = 9;
+      // The transport is manually or automatically parked at a predefined floor and will not provide
+      // any further service. This usually happens in a low traffic condition for the purpose of
+      // energy saving.
+      PARKING = 10;
+      // The transport is returning to its home position, usually the lobby or ground floor.
+      HOMING = 11;
+      // Control of the transport is performed from a control panel in the control cabinet. The transport is not
+      // available for normal operation.
+      CABINET_RECALL = 12;
+      // The transport is under an occupant evacuation mode of control. This usually happens during a
+      // fire alarm when the transport is used for evacuation of occupants.
+      OCCUPANT_EVACUATION = 13;
+      // The transport is operational but is reserved for the transportation of goods only.
+      FREIGHT = 14;
+      FAILURE = 15;
+      REDUCED_SPEED = 16;
+      STORM_OPERATION = 17;
+      HIGH_WIND_OPERATION = 18;
+    }
+
+    enum Alarm {
+      ALARM_UNSPECIFIED = 0;
+      UNACTIVATED = 1;
+      ACTIVATED = 2;
+    }
+
+    enum Active {
+      ACTIVE_UNSPECIFIED = 0;
+      INACTIVE = 1;
```

Comment:
You use the term `idle` in the docs where Active is used. You should probably stay consistent.